client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -73,9 +73,14 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 				</div>
 			</div>
 			{ isDesktopView && ! isDesktopApp && (
-				<Button className="go-mobile__email-link-button" onClick={ emailLogin }>
-					{ translate( 'Email download link' ) }
-				</Button>
+				<div className="go-mobile__email-link">
+					{ translate(
+						'We can email a link to open on your phone to instantly download the app.'
+					) }
+					<Button className="go-mobile__email-link-button" onClick={ emailLogin } borderless>
```

Comment:
Disclaimer: not tested, but can't the `is-link` class just be added? I _think_ that'd minimise the need for additional styling, and is the usual pattern with these kind of buttons. 

```suggestion
					<Button className="go-mobile__email-link-button is-link" onClick={ emailLogin } borderless>
```

Reply:
Right, I think we do have some styling presets @Aurorum  if we do go down that path.